frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -124,7 +128,8 @@ def process_exception(self, request, exception, spider):
     def open(self, spider):
         log.msg('Starting frontier', log.INFO)
         if not self.frontier.manager.auto_start:
-            self.frontier.start()
+            settings_override = getattr(spider, 'frontier_settings', {})
```

Comment:
Sorry for that. I just walked through code again, and found that instantiation in ``open()`` should be cheaper from many prospectives: there is no need to modify frontier APIs and disabling auto start. Unfortunately, that wasn't clear for me since the beginning.